pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -865,6 +868,61 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 				if err != nil {
 					return err
 				}
+				if driverName == "pxd" {
```

Comment:
Do you need to check this everytime we comeback ? Shouldnt this be checked during the validation step before we start any of the stages?